domain/src/gateway/tiptap.rs

```diff
@@ -14,7 +14,7 @@ pub(crate) async fn client(config: &Config) -> Result<reqwest::Client, Error> {
 }
 
 async fn build_auth_headers(config: &Config) -> Result<reqwest::header::HeaderMap, Error> {
-    let auth_key = config.tip_tap_auth_key().ok_or_else(|| {
+    let auth_key = config.tiptap_auth_key().ok_or_else(|| {
```

Comment:
These and a few similar changes are just to match how Tiptap spells their name.

Reply:
This is great, I think this kind of attention to detail helps avoid a lot of bugs and uncertainty in the future.